cmd/cluster/support/delete.go

```diff
@@ -105,7 +104,7 @@ func (o *deleteOptions) run() error {
 		os.Exit(1)
 	}
 
-	deleteRequest, err := createDeleteRequest(connection, cluster, o.limitedSupportReasonID)
+	deleteRequest, err := createDeleteRequest(connection, cluster.ID(), o.limitedSupportReasonID)
```

Comment:
Does this need to be changed to be tested?

Reply:
think leaving this as cluster *v1.Cluster is better here because we KNOW a v1.Cluster has the internal ID as part of it, where as with a string, there's no guarantee about the data passed in.